sample-extension-plugin/build.gradle

```diff
@@ -53,13 +60,19 @@ validateNebulaPom.enabled = false
 testingConventions.enabled = false
 loggerUsageCheck.enabled = false
 
+jacoco {
+    toolVersion = '0.8.13'
+}
+
+check.dependsOn jacocoTestReport
+
 integTest {
     if (project.hasProperty('excludeTests')) {
         project.properties['excludeTests']?.replaceAll('\\s', '')?.split('[,;]')?.each {
             exclude "${it}"
         }
     }
-    systemProperty 'tests.security.manager', 'true'
+    systemProperty 'tests.security.manager', 'false'
```

Comment:
I had to disable this bc the agent was preventing `dumpCoverage` from writing to `build/jacoco`.

I plan to figure out how to re-enable this in a subsequent PR.